contracts/LostAndFound.cdc

```diff
@@ -344,7 +344,7 @@ pub contract LostAndFound {
         }
 
         // Redeem a specific ticket instead of all of a certain type.
-        pub fun redeem(type: Type, ticketID: UInt64, receiver: Capability) {
+        access(Withdraw) fun redeem(type: Type, ticketID: UInt64, receiver: Capability) {
```

Comment:
if redeemAll is access(all), why does this have to be access(Withdraw)

Reply:
Good callout! Fixed